packages/react-native-renderer/src/ReactFabric.js

```diff
@@ -84,7 +83,6 @@ const ReactFabric: ReactNativeType = {
     // Used as a mixin in many createClass-based components
     NativeMethodsMixin,
     // Used by react-native-github/Libraries/ components
-    ReactNativeBridgeEventPlugin, // requireNativeComponent
```

Comment:
We can also delete the forwarding `scripts/rollup/shims/react-native/ReactNativeBridgeEventPlugin.js` now?

Reply:
Should we just get rid of the `ReactNativeBridgeEventPlugin` entirely now and just move `extractEvents` over into the view config registry as well?

Reply:
Good catch on the shim. I'll delete it.